ui/DesignSystem/Primitive/Input/Validated.svelte

```diff
@@ -0,0 +1,108 @@
+<script>
+  import Text from "../Text.svelte";
+  import Icon from "../Icon";
+  import Avatar from "../Avatar.svelte";
+
+  import { ValidationStatus } from "../../../src/validation.ts";
+
+  export let style = null;
+  export let placeholder = null;
+  export let value = null;
+  export let dataCy = null;
```

Comment:
We had a longish disucssion about this on IRC, afaik we want to move away from cypress specific leakage in non-test code. /cc @cloudhead @rudolfs

Reply:
What's the status on whether or not we should use cypress-specific syntax in production code @rudolfs @cloudhead ?

Reply:
👍